resources/state_of_the_dapps.json

```diff
@@ -0,0 +1,43 @@
+{
+    "auction-house":
+    {
+        "name":
+        {
+            "en": "Auction House"
+        },
+        "photo-path": "contacts://auction-house",
+        "state-of-the-dapp?": true,
```

Comment:
I'm not a big fan of ` "state-of-the-dapp?"` i mean the contact list will have groups and @flexsurfer can probably chime in here. so dapp? makes sense because there is a distinction between an average user and a dapp. but we should really have some `"group": "State of the DApps"` or something like this